### PR TITLE
Fix theme loading on earlier versions

### LINKED ISSUE #6534: [Beta] Supervisor tab is not Dark-mode capable
## Checklist

- [x] I have updated to the latest available Home Assistant version.
0.114.0b0

- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
Chrome, Firefox & Edge; HA Android App 2.0.0-235-full

## The problem

grafik

## Expected behavior

   Something that fits to the dark theme 😉

## Steps to reproduce

    - Update to 0.114.0b0
    - Select the supervisor tab

## Environment

- Browser and browser version: Chrome 84.0.4147.105, Firefox 79.0, Edge 84.0.522.52; HA Android App 2.0.0-235-full
- Operating system: HA OS as Proxmox VM

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Fixes theme rendering in the supervisor panel for older versions of Home Assistant.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6534
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/hassio-main.ts [hunk 1]
```diff
@@ -94,7 +95,10 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
     applyThemesOnElement(
```
Comment (bramkragten) [inline hassio/src/hassio-main.ts:95]: Does this function also need some version check or is that ok?

Comment (ludeeus) [reply hassio/src/hassio-main.ts:95]: We have always used that function, so I think this is fine.
This is tested with 1.110 and 114 dev

Comment (bramkragten) [reply hassio/src/hassio-main.ts:95]: But that function was changed pretty heavily, and checks for properties in hass.selectedTheme

Comment (ludeeus) [reply hassio/src/hassio-main.ts:95]: True, but that does not seem to matter.

Tested both versions with:
- Custom theme
- Backend selected
- Default

In all cases this now does what it's expected of it.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**